core/test/ParaGrid_test.cpp

```diff
@@ -420,28 +515,39 @@ TEST_CASE("Test array ordering")
 
     double xFactor = 10;
 
+#ifdef USE_MPI
+    if (test_rank == 0) {
+      ModelArray::setDimension(ModelArray::Dimension::X, nx, 4, 0);
+    }
+    if (test_rank == 1) {
+      ModelArray::setDimension(ModelArray::Dimension::X, nx, 5, 4);
+    }
+    ModelArray::setDimension(ModelArray::Dimension::Y, ny, ny, 0);
+    ModelArray::setDimension(ModelArray::Dimension::Z, NZLevels::get(), NZLevels::get(), 0);
+#else
     ModelArray::setDimension(ModelArray::Dimension::X, nx);
     ModelArray::setDimension(ModelArray::Dimension::Y, ny);
     ModelArray::setDimension(ModelArray::Dimension::Z, NZLevels::get());
+#endif
 
     HField index2d(ModelArray::Type::H);
     index2d.resize();
     std::string fieldName = "index2d";
     std::set<std::string> fields = { fieldName };
     TimePoint time;
 
-    ModelState state = ParaGridIO::readForcingTimeStatic(
-        fields, time, TO_STR(TEST_FILE_SOURCE) + std::string("/") + inputFilename);
+    printf("path = %s\n", inputFilename.c_str());
```

Comment:
How come you used `printf` rather than `std::cout`?

Reply:
I didn't mean to leave this in anyway. It's been removed. It was for debugging purposes.

on a side note though... I use `printf` because it doesn't require additional header files.